src/providers/local/exit-pool.provider.ts

```diff
@@ -97,17 +97,14 @@ export const exitPoolProvider = (
   const { txState, txInProgress } = useTxState();
   const { transactionDeadline } = useApp();
   const { slippageBsp } = useUserSettings();
-  const { getSigner } = useWeb3();
+  const { account, getSigner } = useWeb3();
   const { relayerSignature, relayerApprovalAction, relayerApprovalTx } =
     useRelayerApproval(RelayerType.BATCH);
 
   const debounceQueryExit = debounce(queryExit, debounceQueryExitMillis);
   const debounceGetSingleAssetMax = debounce(
     getSingleAssetMax,
-    debounceGetSingleAssetMaxMillis,
-    {
-      leading: true,
```

Comment:
Why was this needed before but isn't needed now?

Reply:
I couldn't figure that out, I removed it to see if it would fix the problem. Let me double-check if this needs to be removed.

Reply:
Yeah, this wasn't related to the issue, so I've reverted.